scheme/cca-ssd-platform/evidence_handler.go

```diff
@@ -46,13 +46,17 @@ func (s EvidenceHandler) SynthKeysFromTrustAnchor(tenantID string, ta *handler.E
 	return arm.SynthKeysFromTrustAnchors(SchemeName, tenantID, ta)
 }
 
-func (s EvidenceHandler) GetTrustAnchorID(token *proto.AttestationToken) (string, error) {
-	return arm.GetTrustAnchorID(SchemeName, token)
+func (s EvidenceHandler) GetTrustAnchorIDs(token *proto.AttestationToken) ([]string, error) {
+	ta, err := arm.GetTrustAnchorID(SchemeName, token)
+	if err != nil {
+		return nil, err
```

Comment:
not sure this has any impact (we'd need to check all the calling sites) but there is a slight inconsistency here: we return `<nil>` when in other similar cases (e.g., see `handler/evidence_rpc.go`) we return `[]string{""}` instead.

Reply:
Agree, I have made it consistent! Thanks for pointing this out!